okada_html/home.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>dobocreate株式会社</title>
+    <link rel="stylesheet" href="dobocreate.css" />
+    <style>
+        </style>
+  </head>
+  <body>
+    <header>
+            <div class="logo">
+                <img src="img/ドボクリエイト0-e1626942215397.jpg"alt="ロゴ" width=250px></div>
+                <ul>ホーム</ul> 
+                <ul>会社情報</ul> 
+                <ul>採用情報</ul> 
+                <ul>事業内容</ul> 
+                <ul>発表論文 </ul> 
+                <ul>アクセス</ul> 
+                <ul>お問い合わせ</ul> 
+                <ul>Instagram</ul> 
+    </header>
+    <section>
+        <div class="greeting">
+            <h1 >ごあいさつ</h1>
+            <p>「社会がちょっと変わる」そんな挑戦をしていきたいという想いを込め、
+                「ドボク」の新しい価値・見える化を「クリエイト」する会社を創業し、
+                土木業界のi-constructionやICTとともに,私たちも成長してまいりました。
+                <br><br>
+                これからも、数値解析やAIなどを駆使し、DXというアプローチでインフラ・
+                レジデンスの向上に貢献すべく、研鑽してまいります。このスタートライン
+                に立てたのもたくさんの出会いに支えられ、めぐり合ってきた方々のおかげ
+                です。めぐり逢いを大切にし、一歩でも目標に近づけるように努力を重ねて
+                いきますので、これかたの応援よろしくお願い致します。
+            <br><br>
+            ぜひ、私たちドボクリエイトにご期待ください</p>
```

Comment:
後述していますが、子要素が複数にまたがる場合には、開始タグと終了タグを改行させて、インデントをそろえてください。
```suggestion
            <p>
               「社会がちょっと変わる」そんな挑戦をしていきたいという想いを込め、
                「ドボク」の新しい価値・見える化を「クリエイト」する会社を創業し、
                土木業界のi-constructionやICTとともに,私たちも成長してまいりました。
                <br><br>
                これからも、数値解析やAIなどを駆使し、DXというアプローチでインフラ・
                レジデンスの向上に貢献すべく、研鑽してまいります。このスタートライン
                に立てたのもたくさんの出会いに支えられ、めぐり合ってきた方々のおかげ
                です。めぐり逢いを大切にし、一歩でも目標に近づけるように努力を重ねて
                いきますので、これかたの応援よろしくお願い致します。
                <br><br>
                ぜひ、私たちドボクリエイトにご期待ください
           </p>
```